src/frontend/admin/src/components/templates/contract-definition/inputs/ContractDefinitionSearch.tsx

```diff
@@ -40,7 +40,7 @@ export function ContractDefinitionSearch(
         items={contractDefinitionsQuery.items}
         loading={contractDefinitionsQuery.states.fetching}
         onAddClick={onAddClick}
-        onEditClick={() => modal.handleOpen()}
+        onEditClick={modal.handleOpen}
         onFilter={(term) => setQuery(term)}
```

Comment:
If you do that you can also do : 

```suggestion
        onEditClick={modal.handleOpen}
        onFilter={setQuery}
```